app/styles/_openshift-icon.less

```diff
@@ -1,331 +0,0 @@
-  content: "\e031";
-}
-.icon-system-signal-lg:before {
-  content: "\e032";
-}
-.icon-system-lg:before {
-  content: "\e033";
-}
-.icon-calendar:before {
-  content: "\ee00";
-}
-.icon-warning-sign:before {
-  content: "\ee01";
-}
-.icon-question-sign:before {
-  content: "\ee02";
-}
-.icon-search:before {
-  content: "\ee03";
-}
-.icon-twitter:before {
-  content: "\ee04";
-}
-.icon-facebook:before {
-  content: "\ee05";
-}
-.icon-google-plus:before {
-  content: "\ee06";
-}
-.icon-info-sign:before {
-  content: "\ee07";
-}
-.icon-bubbles:before {
-  content: "\ee08";
-}
-.icon-code:before {
-  content: "\ee10";
-}
-.icon-angle-down:before {
-  content: "\ee12";
-}
-.icon-angle-up:before {
-  content: "\ee13";
-}
-.icon-angle-right:before {
-  content: "\ee14";
-}
-.icon-angle-left:before {
-  content: "\ee15";
-}
-.icon-plus:before {
-  content: "\ee16";
-}
-.icon-minus:before {
-  content: "\ee17";
-}
-.icon-pencil:before {
-  content: "\ee18";
-}
-.icon-remove-circle:before {
-  content: "\ee19";
-}
-.icon-ban-circle:before {
-  content: "\ee20";
-}
-.icon-ok:before {
-  content: "\ee21";
-}
-.icon-remove:before {
-  content: "\ee22";
-}
-.icon-check:before {
-  content: "\ee23";
-}
-.icon-check-empty:before {
-  content: "\ee24";
-}
-.icon-cloud-upload:before {
-  content: "\ee25";
-}
-.icon-cloud-download:before {
-  content: "\ee26";
-}
-.icon-cloud:before {
-  content: "\ee27";
-}
-.icon-cogs:before {
-  content: "\ee28";
-}
-.icon-download-alt:before {
-  content: "\ee29";
-}
-.icon-bolt:before {
-  content: "\ee30";
-}
-.icon-star-empty:before {
-  content: "\ee31";
-}
-.icon-time:before {
-  content: "\ee32";
-}
-.icon-busy:before {
-  content: "\ee33";
-}
-.icon-list-ol:before {
-  content: "\ee34";
-}
-.icon-app-code:before {
-  content: "\ee35";
-}
-.icon-user:before {
-  content: "\ee36";
-}
-.icon-users:before {
-  content: "\ee37";
-}
-.icon-colocation2:before {
-  content: "\ee38";
-}
-.icon-tasks:before {
-  content: "\ee39";
-}
-.icon-equalizer:before {
-  content: "\ee40";
-}
-.icon-dollar:before {
-  content: "\ee41";
-}
-.icon-euro:before {
-  content: "\ee42";
-}
-.icon-credit:before {
-  content: "\ee43";
-}
-.icon-bug:before {
-  content: "\ee44";
-}
-.icon-domain2:before {
-  content: "\ee45";
-}
-.icon-dyi-hammer:before {
-  content: "\ee46";
-}
-.icon-copy-clipboard:before {
-  content: "\ee47";
-}
-.icon-app-box:before {
-  content: "\ee48";
-}
-.icon-database:before {
-  content: "\ee49";
-}
-.icon-shield:before {
-  content: "\ee50";
-}
-.icon-cartridge-revised:before {
-  content: "\ee51";
-}
-.icon-cartridge-cloud:before {
-  content: "\ee52";
-}
-.icon-cartridge-cloud2:before {
-  content: "\ee53";
-}
-.icon-cartridge-custom:before {
-  content: "\ee54";
-}
-
-
-// needed for dual icon alignment
-.font-icon-block {
-  display: inline-block;
-  position: relative;
-  .font-icon:nth-child(n+2) {
-    position: absolute;
-    left: 0;
-    top: 0;
-    text-shadow: none;
-  }
-}
-
-.font-icon {
```

Comment:
Nope.  I tested a couple context where we use .font-icons from logos (existing catalog and service catalog) and everything is fine without those styles.